src/vs/base/browser/ui/inputbox/inputBox.ts

```diff
@@ -520,9 +520,13 @@ export class HistoryInputBox extends InputBox implements IHistoryNavigationWidge
 
 	private readonly history: HistoryNavigator<string>;
 
+	private readonly _onInput = this._register(new Emitter<void>());
+	public readonly onInput: Event<void> = this._onInput.event;
```

Comment:
What's wrong with "onDidChange"? It's been used in numerous places.

Reply:
Did not see that, my mistake.. fixed in latest commit.